### PR TITLE
feat: Update mimeType extensions from mime-db v1.52.0

### PR DESCRIPTION
**Pull request by franznoel**
## Description

Updated the `mimeTypes` list of `system` coming from mime-db v1.52.0

Taken from this list:  https://github.com/jshttp/mime-db/blob/v1.52.0/db.json

### CODE REVIEW

File: test/system.spec.ts [hunk 1]
```diff
@@ -2,84 +2,86 @@ import { afterEach, describe, expect, it } from 'vitest';
...
-  'semver',
-];
-
 describe('system', () => {
   afterEach(() => {
     faker.locale = 'en';
   });
 
+  // TODO @prisis 2022-01-31: Add multiple seed based expectations.
+  const seededRuns = [
```
Comment (ST-DDT) [inline test/system.spec.ts:13]: Why did you move these?

Comment (franznoel) [reply test/system.spec.ts:13]: I accidentally moved it. Will move it back

**Comment by ST-DDT** [comment]
Please also update the version number in the jsdocs.

**Comment by franznoel** [comment]
> update the version number

How should I update the jsdocs version?

**Comment by ST-DDT** [comment]
> > update the version number
> 
> How should I update the jsdocs version?

I'm referring to this line: https://github.com/faker-js/faker/blob/1058e17ebb24207a5aad959b5613cc2d1de849d7/src/locales/en/system/mimeTypes.ts#L25

Please update it to the exact tag-name (version) of the commit you used to import the db from.

I assume it's https://github.com/jshttp/mime-db/releases/tag/v1.52.0 .

**Review by ST-DDT — APPROVED** [review]

File: src/locales/en/system/mimeTypes.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 
-Definitions from mime-db v1.21.0
+Definitions from mime-db v1.52.0
 For updates check: https://github.com/jshttp/mime-db/blob/master/db.json
```
Comment (Shinigami92) [inline src/locales/en/system/mimeTypes.ts]: Maybe we should use a more strict link here: https://github.com/jshttp/mime-db/blob/v1.52.0/db.json

File: test/system.spec.ts [hunk 3]
```diff
@@ -100,14 +100,22 @@ describe('system', () => {
             'mpeg',
             'wav',
             'png',
+            'jpe',
```
Comment (Shinigami92) [inline test/system.spec.ts:103]: Is this a typo?

Comment (ST-DDT) [reply test/system.spec.ts:103]: It's present in the mime types (although I haven't seen or used it either).

Comment (franznoel) [reply test/system.spec.ts:103]: Here's the line number.
https://github.com/faker-js/faker/pull/535/files#diff-173ae4ba4f5ef9f577fb7f4812ad92a6d0c81a82db626bec5b690c32f4e24bb1L5507

**Review by ST-DDT — APPROVED** [review]

**Comment by HonzaMac** [comment]
I would love to see this out in next version, how can I help with this?

**Comment by ST-DDT** [comment]
@Shinigami92 IMO this is a `c: locale` and thus can be merged in v6.1.

**Review by ST-DDT — APPROVED** [review]

**Comment by Shinigami92** [comment]
> @Shinigami92 IMO this is a `c: locale` and thus can be merged in v6.1.

All `c: locale` are currently target for `v6.2`: https://github.com/faker-js/faker/pulls?q=is%3Aopen+is%3Apr+label%3A%22c%3A+locale%22
Why handle this PR differently?

**Comment by HonzaMac** [comment]
We use ´mime-types´ for input validation (which use mimedb jnside). Problem is that faker generates extensions i.e. ´mbp´ which was renamed later, but it fails our suites sometimes when Faker generate it. Faker have mimedb from 2016.

**Comment by ST-DDT** [comment]
This PR needs to be rebased/updated.

**Comment by prisis** [comment]
I have a different approach #808

**Comment by ST-DDT** [comment]
Superseded by #808 . Thanks for your contribution.
